service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala

```diff
@@ -336,4 +336,20 @@ class DefaultMahaCalciteSqlParserTest extends BaseMahaServiceTest with Matchers
 
     assert(request.filterExpressions.toString contains "EqualityFilter(Student ID,123,false,false)")
   }
+
+  test("test filter: like") {
+
+    val sql =
+      s"""
+              select * from student_performance
+              where 'Student ID' like '123%'
```

Comment:
can you add test for wrapping value in ""

Reply:
added